### PR TITLE
use last known controller position and rotation if tracking fails

### LINKED ISSUE #2: Error when controller is not tracking
Hey,
if the vive controller is not tracked anymore an error occurs:
Uncaught TypeError: Cannot read property '0' of null...

Position and Rotation of the pose are null.

I would like to fix this. My approach would be to save the last available values and use these if the tracking fails instead.

### PR DESCRIPTION
**Pull request by valentin-pinkau**
fixes #2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -39,13 +39,17 @@ module.exports = function(THREE, packageRoot) {
...
+        var lastPoseOrientation = [0, 0, 0, 1]
+
         var vivePath = packageRoot + 'assets/vr_controller_vive_1_5.obj'
         var loader = new THREE.OBJLoader()
         loader.load(vivePath, function(object) {
             var loader = new THREE.TextureLoader()
             model = object.children[0]
             model.material.map = loader.load(packageRoot + 'assets/onepointfive_texture.png')
             model.material.specularMap = loader.load(packageRoot + 'assets/onepointfive_spec.png')
+            model.material.color = new THREE.Color(1, 1, 1)
```
Comment (msfeldstein) [inline index.js]: Is this needed since it was changed in the other PR?

**Review by msfeldstein — COMMENTED** [review]
Do you think this is the right thing to do?  With this solution apps won't know if the controller is tracking, or if it's still seeing updates.  There should be some way to discover that the controller is no longer available so the app developers can do some error handling.

**Comment by valentin-pinkau** [comment]
Fair enough. I would suggest then to set a flag, that the controller is not tracking. App developers are then able to catch this and do their own error handling and overwrite our defaults or decide to not render the controller or ...

**Comment by msfeldstein** [comment]
v cool thanks!

**Comment by msfeldstein** [comment]
just pushed in v1.1.0

**Comment by valentin-pinkau** [comment]
great :)

**Pull request merged**